docs/mapping-annotations-for-AreaC.md

```diff
@@ -0,0 +1,200 @@
+  - It overrides `m_def` if both exist.
+- `m_def` (on the section definition):
+  - Use for a default that should work anywhere that section appears. Example: `Simulation.m_def` with `@` to anchor the whole tree.
+  - Use to give a subsection its own base path so children can resolve relative paths. Example: `DFT.m_def` with `.dft`, so all quantities inside DFT can start from `.dft` even if the parent doesn’t have a special annotation.
+  - Use for abstract/base sections when you need a mapping that applies to all their concrete inheritors.
+- Lookup order: attribute annotation → `m_def` annotation → inheriting section definitions (for abstract subsections). If you want a local override, put it on the attribute; if you want a reusable default or a base path, put it on `m_def`.
+
+## Core patterns 
+
+### 1) Root anchor
+Anchor the target root section to the **source tree root** (`@` = top of source dict/XML/HDF5).
+```python
+add_mapping_annotation(general.Simulation.m_def, OUT_KEY, '@')
+```
+
+### 2) Nested quantities
+Relative to the parent section’s mapped node.
+```python
+add_mapping_annotation(general.Program.name, OUT_KEY, '.program')
+add_mapping_annotation(general.Program.version, OUT_KEY, '.program_version')
+```
+
+### 3) Single nested subsection
+Annotate the subsection’s section definition.
+```python
+add_mapping_annotation(model_method.DFT.m_def, OUT_KEY, '.dft')
+```
+
+### 4) Repeating subsections (lists)
+Point to a list; one subsection is created per element. For repeating quantities or array-like quantities, map to a list/array; it is set directly.
+```python
+add_mapping_annotation(model_system.AtomsState.m_def, OUT_KEY, '.atoms')
+add_mapping_annotation(model_system.AtomsState.chemical_symbol, OUT_KEY, '.label')
+add_mapping_annotation(model_system.AtomsState.atomic_number, OUT_KEY, '.number')
+add_mapping_annotation(model_system.ModelSystem.positions, OUT_KEY, '.positions')  # shape [n_atoms, 3]
+```
+
+### 5) Multiple sources into the same schema
+Use different keys; run `convert` once per key.
+```python
+# Octopus uses OUT_KEY + INFO_KEY + EIGENVALUES_KEY on the same schema
+# Exciting uses INFO_KEY, INPUT_XML_KEY, EIGVAL_KEY, BANDSTRUCTURE_XML_KEY, DOS_XML_KEY
+# Wannier90 uses WOUT_KEY, WIN_KEY, BAND_KEY, WHR_KEY, DOS_KEY
+```
+Set `annotation_key = <KEY>` before each `convert`.
+
+### 6) Transformations (reshape, filter, apply units)
+Use a tuple `(function_name, [paths], {kwargs})` and implement the function on the parser.
+```python
+add_mapping_annotation(
+    outputs.TotalForce.value,
+    SECONDARY_KEY,
+    ('reshape_forces', ['.raw_forces'], {'unit': 'eV / angstrom'})
+)
+```
+
+
+### 7) Units or subfield search
+You can pass `unit='eV'` or `search='.path.to.value'` in the annotation (see Wannier90 eigenvalue/DOS paths).
+
+## Starter template
+
+File: `packages/nomad_simulation_parsers/schema_packages/template.py`
```

Comment:
What is the purpose of this line? Regardless, given the size of this template, I would move it out to exactly that file path. You should of course reference it from these docs.